docs/src/components/pricing/PricingWhatToExpect.tsx

```diff
@@ -0,0 +1,207 @@
+                  Maintenance and support
+                </Typography>
+              </Box>
+              <Typography variant="body2" color="text.secondary">
+                With your purchase, you receive support and access to new versions for the duration
+                of your subscription. You can{' '}
+                <Link href="https://mui.com/x/introduction/support/#technical-support">
+                  learn more about support
+                </Link>{' '}
+                in the docs. Note that, except for critical issues, such as security flaws, we
+                release bug fixes and other improvements on top of the latest version, instead of
+                patching older versions.
+              </Typography>
+            </Paper>
+          </Grid>
+          <Grid item xs={12} sm={6}>
+            <Paper variant="outlined" sx={{ p: 2, height: '100%' }}>
+              <Box sx={{ display: 'flex', alignItems: 'center', mb: 1 }}>
+                <AllInclusiveOutlinedIcon fontSize="small" color="primary" />
+                <Typography
+                  fontWeight="bold"
+                  component="h3"
+                  color="text.primary"
+                  variant="body2"
+                  sx={{ ml: 1 }}
+                >
+                  Perpetual vs. Annual license type
+                </Typography>
+              </Box>
+              <Typography variant="body2" color="text.secondary">
+                On both license types, any version released before the end of your license term is
+                forever available for applications deployed in production.
+                <br />
+                The difference regards the right to use the components for development purposes.
+                Only the perpetual license type allows you to continue development once your license
+                expires.
+              </Typography>
+            </Paper>
+          </Grid>
+          <Grid item xs={12} sm={6}>
+            <Paper variant="outlined" sx={{ p: 2, height: '100%' }}>
+              <Box sx={{ display: 'flex', alignItems: 'center', mb: 1 }}>
+                <ReplayRoundedIcon fontSize="small" color="primary" />
+                <Typography
+                  fontWeight="bold"
+                  component="h3"
+                  color="text.primary"
+                  variant="body2"
+                  sx={{ ml: 1 }}
+                >
+                  Annual license type
+                </Typography>
+              </Box>
+              <Typography variant="body2" color="text.secondary">
+                The Annual license type requires an active license to use the software in
+                development. You will need to renew your license if you wish to continue active
+                development after your current license term expires.
+                <br />
+                <br />
+                The license is perpetual in production so you {"don't"} need to renew your license
```

Comment:
I don't understand what the notation {"word"} is supposed to do.

Perhaps we can emphasize it with italic?
```suggestion
                The license is perpetual in production, so you *don't* need to renew your license
```

Reply:
It's to fix an eslint/React quote issue